Modules/Sources/JetpackStats/Services/Data/SiteMetric.swift

```diff
@@ -0,0 +1,84 @@
+import SwiftUI
+
+enum SiteMetric: String, CaseIterable, Identifiable, Sendable, Codable {
+    case views
+    case visitors
+    case likes
+    case comments
+    case posts
+    case timeOnSite
+    case bounceRate
```

Comment:
Is this bounce rate metric shown in the new Stats? If not, can it be deleted?

Reply:
These are supposed to be added in the near future. We've already added localizable strings and everything. I'd usually be in favor of removing the unused code, but I'd rather keep it as it's just the definitions.